packages/build/src/plugins_core/blobs_upload/index.ts

```diff
@@ -47,13 +47,6 @@ const coreStep: CoreStepFunction = async function ({
     return {}
   }
 
-  // If using the deploy config API or the Frameworks API, configure the store
-  // to use the region that was configured for the deploy. We don't do it for
-  // the legacy file-based upload API since that would be a breaking change.
-  if (blobs.apiVersion > 1) {
-    storeOpts.experimentalRegion = 'auto'
-  }
-
```

Comment:
Don't we need to specify reverse condition there by specifying some region for the "else" condition to not cause breaking change this comment is mentioning?

Reply:
From https://github.com/netlify/blobs/pull/183

> Should you want to update your Blobs client but keep accessing a deploy-scoped store that has been created with an older version, you can use getDeployStore({ region: "us-east-2" }) to ensure you're using the right region.

so something like this:
```
if (blobs.apiVersion === 1) {
  storeOpts.region = 'us-east-2'
}
```
?